tools/trim_adapters/trim_adapters.xml

```diff
@@ -0,0 +1,71 @@
+<tool id="trim_adapters" name="Trim putative adapter sequence" version="0.1.0">
+    <requirements>
+        <requirement type="package" version="0.1.6">atactk</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        ln -s '$input1' ./forward.fastq &&
+        ln -s '$input2' ./reverse.fastq &&
+        trim_adapters
+            #if '$settings.advanced' == 'advanced'
+                -d '$settings.edit_distance'
+                -f '$settings.fudge'
+                -s '$settings.trim_start'
+                -r '$settings.rc_length'
+            #end if
+            forward.fastq reverse.fastq
+    ]]></command>
+    <inputs>
+        <param type="data" name="input1" format="fastq,fastq.gz" />
+        <param type="data" name="input2" format="fastq,fastq.gz" />
+        <conditional name="settings">
+            <param name="advanced" type="select" label="Specify advanced parameters">
+                <option value="simple" selected="true">No, use program defaults.</option>
+                <option value="advanced">Yes, see full parameter list.</option>
+            </param>
+            <when value="simple"></when>
+            <when value="advanced">
+                <param name="edit_distance" label="The maximum edit distance permitted when aligning the paired reads" type="integer" min="1" value="1" help="(-d)" />
+                <param name="fudge" label="An arbitrary number of extra bases to trim from the ends of reads" type="integer" min="1" value="1" help="(-f)" />
+                <param name="trim_start" label="Trim this number of bases from the start of each sequence" type="integer" min="0" value="0" help="(-s)"/>
+                <param name="rc_length" label="Use the reverse complement of this number of from the beginning of the reverse read to align reads" type="integer" min="1" value="20" help="(-r)"/>
+            </when>
+        </conditional>
+    </inputs>
+    <outputs>
+        <data name="output1" format="fastq.gz" from_work_dir="forward.trimmed.fastq.gz" />
+        <data name="output2" format="fastq.gz" from_work_dir="reverse.trimmed.fastq.gz" />
+    </outputs>
+    <tests>
+        <test>
+            <param name="input1" value="SP1_f.fastq" ftype="fastq" />
+            <param name="input2" value="SP1_r.fastq" ftype="fastq" />
+            <output name="output1" file="SP1_f.trimmed.fastq.gz" decompress="True"/>
+            <output name="output2" file="SP1_r.trimmed.fastq.gz" decompress="True"/>
+        </test>
+    </tests>
+    <help><![CDATA[
+usage:
```

Comment:
Can you follow this one here: https://github.com/bgruening/galaxytools/blob/master/tools/augustus/augustus.xml#L258

What is does: and so on ... you can test the look with http://rst.ninjs.org/